### PR TITLE
Strip trailing comma from metadata

### PR DESCRIPTION
**Pull request by zmwangx**
Without this patch we could see awkward output like

```
  3.  iNEWS FTP/FTPS Server Specification - Avid
       http://resources.avid.com/SupportFiles/attach/Broadcast/iNEWS-FTP-FTPS-ServerProtocol.pdf
       Sep 25, 2017,
       FTP (File Transfer Protocol) interaction. The iNEWS FTP Server supports a subset of the FTP commands described in the FTP specification and ...
```

### CODE REVIEW

File: googler [hunk 1]
```diff
@@ -2235,6 +2235,8 @@ class GoogleParser(object):
                 try:
                     metadata = div_g.select('.f').text
                     metadata = metadata.replace('\u200e', '').replace(' - ', ', ').strip()
+                    if metadata.endswith(','):
+                        metadata = metadata.rstrip(',')
```
Comment (jarun) [inline googler:2239]: What happens if we just `metadata = metadata.rstrip(',')` without the check? If there's no `,` at the end there would be no change to metadata, right?

Comment (zmwangx) [reply googler:2239]: Yeah, this is what happens when I write down the first line without even thinking about what should come next.

Comment (zmwangx) [reply googler]: Fixed.

Comment (jarun) [reply googler]: It's OK. :+1:

**Comment by jarun** [comment]
Thank you!

**Pull request merged**